tests/QMLEngine/basic.js

```diff
@@ -29,4 +29,9 @@ describe('QMLEngine.basic', function() {
     expect(qml.inner3).toBe(qml.current + 'foo/foo/lol/');
     expect(qml.full).toBe('http://example.com/bar');
   });
+
+  it('the signal disconnect in the signal handler with many subscribers', function() {
```

Comment:
Of course it could. Please suggest and apply anything... (cause it shortest
what my brain produced)

С уважением,
Павел Васёв

2016-07-10 20:55 GMT+05:00 Сковорода Никита Андреевич <
notifications@github.com>:

> In tests/QMLEngine/basic.js
> <https://github.com/qmlweb/qmlweb/pull/291#discussion_r70184097>:
>
> > @@ -29,4 +29,9 @@ describe('QMLEngine.basic', function() {
> >      expect(qml.inner3).toBe(qml.current + 'foo/foo/lol/');
> >      expect(qml.full).toBe('http://example.com/bar');
> >    });
> > +
> > +  it('the signal disconnect in the signal handler with many subscribers', function() {
>
> Could this have a bit shorter name with this explanation (i.e. what it
> does) placed into a comment inside the qml file?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qmlweb/qmlweb/pull/291/files/dcfdfe15e540cb6f8841a617e930f106a9eee5d4#r70184097>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJtyxPVc4t_4yudtSNqeGrMmmgjkdEiks5qURYNgaJpZM4JFWum>
> .
>